examples/lstm_stateful.py

```diff
@@ -154,7 +154,7 @@ def create_model(stateful: bool):
 
 
 # split train/test data
-def split_data(x, y, ratio: int = 0.8):
+def split_data(x, y, ratio: float = 0.8):
```

Comment:
Do we really need a type annotation here, though? `ratio=0.8` sounds like it would be fine...

Reply:
indeed.  I removed it.